### PR TITLE
add hidden directive

### PR DESCRIPTION
**Pull request by tszyuloveyou**
We have selected, checked, readonly and disabled for html input but have no hidden, so, I want to add hidden

### CODE REVIEW

File: src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php [hunk 1]
```diff
@@ -406,4 +406,15 @@ protected function compileEndPushIf()
     {
         return '<?php $__env->stopPush(); endif; ?>';
     }
+
+    /**
+     * Compile a readonly block into valid PHP.
```
Comment (rodrigopedra) [inline src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php:411]: I believe you should update the docblock's description

Comment (rodrigopedra) [reply src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php]: Looks go to me, thanks

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If applicable, please consider releasing your code as a package so that the community can still take advantage of your contributions!

**Comment by tszyuloveyou** [comment]
@taylorotwell if you want to dalay that why you need to close the PR, you don't merge I can understand but I don't understand why you need to close the PR, anyway, up to you, I create to PR that has been fulfill my obligation, I have no obligation to do more.
you don't merge, I also can on AppServiceProvider::boot add Blade::directive to let it work, so, I don't care

**Pull request closed without merging**